presto-docs/src/main/sphinx/admin/queue.rst

```diff
@@ -7,12 +7,18 @@ that can be submitted to Presto and the quota of running queries per queue.
 The filename of the JSON config file should be specified in ``query.queue-config-file``
 config property.
 
-Rules that specify multiple queues will cause the query to enter the queues sequentially.
-Rules are processed sequentially and the first one that matches will be used.
-In the example configuration below, there are five queue templates. In the
-``user.${USER}`` queue, ``${USER}`` will be expanded to the name of the user
-that submitted the query. ``${SOURCE}`` is also supported, which expands to the
-source submitting the query.
+Rules that specify multiple queues will cause the query to acquire the queues' permits sequentially.
+The query must acquire all queues' permits before it starts being executed. It acquires the
+next queue permit only after it is accepted for execution by the previous queue. However
```

Comment:
I don't think the sentence starting with "However" is necessary, it's a bit confusing. If you have it, it should read "However, a slot for the query is reserved in all queues."